enterprise/frontend/src/embedding/data-picker/DataSelector/tests/DataSourceSelector.unit.spec.tsx

```diff
@@ -0,0 +1,185 @@
+import userEvent from "@testing-library/user-event";
```

Comment:
I initially wanted to add this to `DataSelector.unit.spec.js`, but it was a JS file, so the type wasn't working. I then create a new TS file instead.